### PR TITLE
THE SKELETON IS AT HOME

### PR DESCRIPTION
**Pull request by Dman757**
SEE TITLE

### CODE REVIEW

File: src/tenyksscripts/scripts/jnerulastimezone.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 def run(data, settings):
     if data['payload'] == 'jnerula time':
-        tz = pytz.timezone('Asia/Bangkok')
+        tz = pytz.timezone('America/Los_Angeles')
```
Comment (lykkin) [inline src/tenyksscripts/scripts/jnerulastimezone.py:8]: he actually lives in pdx. plz fix

Comment (Dman757) [reply src/tenyksscripts/scripts/jnerulastimezone.py:8]:
```suggestion
        pdx = ''.join(map(chr, filter(None, [65,109,101,114,105,99,97,47,76,111,115,95,65,110,103,101,108,101,115])))
        tz = pytz.timezone(pdx)
```
Thanks for the feedback. Does this fix work for you?

**Comment by Dman757** [comment]
@lykkin CHANGES WERE MADE

**Comment by qpfiffer** [comment]
WHAT ARE YOU TRYING TO HIDE

**Comment by Dman757** [comment]
> WHAT ARE YOU TRYING TO HIDE

Move along sir, this doesn't involve you.

**Comment by Dman757** [comment]
@lykkin pls sir, I need this merged to feed my family

**Pull request merged**